cmd/juju/application/deploy_test.go

```diff
@@ -3149,28 +3053,25 @@ func withCharmRepoResolvable(
 	// resolve that to the real URL, it it may be queried with the fully
 	// qualified URL, or one without series set etc.
 	resultURL := *url
-	if resultURL.Revision < 0 {
```

Comment:
the method `withCharmRepoResolvable` makes me think it's charmstore specific. if so, it can be removed from tests.

Reply:
I took out the charmstore specific parts, most of the time it's actually useful, so I'll keep it :+1: